main.go

```diff
@@ -6,10 +6,11 @@ import (
 
 	"github.com/abs-lang/abs/install"
 	"github.com/abs-lang/abs/repl"
+	"github.com/abs-lang/abs/util"
 )
 
 // Version of the ABS interpreter
-var Version = "dev"
```

Comment:
we shouldn't change this -- it's a [build-time variable](https://github.com/abs-lang/abs/blob/5e90b8845d37524bba2f78794cdeb83702b341d3/scripts/release.abs#L87)

Reply:
OK, will revert that.